django/utils/crypto.py

```diff
@@ -74,3 +74,22 @@ def pbkdf2(password, salt, iterations, dklen=0, digest=None):
     password = force_bytes(password)
     salt = force_bytes(salt)
     return hashlib.pbkdf2_hmac(digest().name, password, salt, iterations, dklen)
+
+
+try:
+    _ = hashlib.md5(usedforsecurity=False)  # nosec
+
+    def md5(string=b'', usedforsecurity=True):
```

Comment:
Is there any reason you can't do the following?

```python
md5 = hashlib.md5
```

Also, it may be cleaner to do this in an `else` of the `try`.

Reply:
Done